### PR TITLE
[M] Added the ImportRecordDTO and ImportUpstreamConsumerDTO

### PR DESCRIPTION
**Pull request by Ceiu**
- Added the new ImportRecordDTO and ImportUpstreamConsumerDTO
- Updated OwnerResource to use the ImportRecordDTO where appropriate

### CODE REVIEW

File: server/src/main/java/org/candlepin/dto/api/v1/ImportRecordTranslator.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    public ImportRecordDTO translate(ModelTranslator translator, ImportRecord source) {
+        return source != null ? this.populate(translator, source, new ImportRecordDTO()) : null;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ImportRecordDTO populate(ImportRecord source, ImportRecordDTO dest) {
+
```
Comment (vritant) [inline server/src/main/java/org/candlepin/dto/api/v1/ImportRecordTranslator.java]: *could* remove whitespace

File: server/src/main/java/org/candlepin/dto/api/v1/ImportUpstreamConsumerDTO.java [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+import org.candlepin.dto.TimestampedCandlepinDTO;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+
+
+/**
+ * A DTO representation of an UpstreamConsumer received during import
```
Comment (vritant) [inline server/src/main/java/org/candlepin/dto/api/v1/ImportUpstreamConsumerDTO.java]: comment needs revisiting

File: server/src/test/java/org/candlepin/dto/api/v1/ImportRecordTranslatorTest.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import org.candlepin.model.ImportUpstreamConsumer;
+
+import org.junit.runner.RunWith;
+
+import java.util.Date;
+
+
+
+/**
+ * Test suite for the CertificateTranslator class
```
Comment (vritant) [inline server/src/test/java/org/candlepin/dto/api/v1/ImportRecordTranslatorTest.java]: comment needs revisiting

File: server/src/main/java/org/candlepin/dto/api/v1/ImportUpstreamConsumerDTO.java (hunk 2 above)
Comment (Ceiu) [reply server/src/main/java/org/candlepin/dto/api/v1/ImportUpstreamConsumerDTO.java]: What needs revisiting? That's exactly what this DTO is.

File: server/src/test/java/org/candlepin/dto/api/v1/ImportRecordTranslatorTest.java (hunk 3 above)
Comment (Ceiu) [reply server/src/test/java/org/candlepin/dto/api/v1/ImportRecordTranslatorTest.java]: Fixed

**Comment by Ceiu** [comment]
@candlepin-pull-request-bot retest this please

File: server/src/main/java/org/candlepin/dto/api/v1/ImportUpstreamConsumerDTO.java (hunk 2 above)
Comment (vritant) [reply server/src/main/java/org/candlepin/dto/api/v1/ImportUpstreamConsumerDTO.java]: ah I misread. ignore please.

**Pull request merged**